addons/mail/static/src/discuss/core/public/discuss_core_public_service.js

```diff
@@ -0,0 +1,38 @@
+/* @odoo-module */
+
+import { reactive } from "@odoo/owl";
+
+import { registry } from "@web/core/registry";
+
+export class DiscussCorePublic {
+    /**
+     * @param {import("@web/env").OdooEnv} env
+     * @param {{
+     * bus_service: ReturnType<typeof import("@bus/services/bus_service").busService.start>,
+     * }} services
+     */
```

Comment:
Inline `@type` similar to others instead? Or the other way around, but to be consistent.